Makefile

```diff
@@ -40,22 +40,22 @@ test-unit:
 	pipenv run ./scripts/run_tests_unit.sh
 
 test-functional: generate-pages
-	yarn test_functional
+	npm run test_functional
 
 test-functional-headless: generate-pages
 	EQ_RUN_FUNCTIONAL_TESTS_HEADLESS='True' make test-functional
 
 test-functional-spec: generate-pages
-	yarn test_functional --spec ./tests/functional/spec/$(SPEC)
+	npm run test_functional -- --spec=./tests/functional/spec/$(SPEC)
 
 test-functional-suite: generate-pages
-	yarn test_functional --suite $(SUITE)
+	npm run test_functional -- --suite=$(SUITE)
```

Comment:
Why do have the extra `--` lines?

Reply:
Without the extra `--` it doesn't run a specified spec or suite, it will run all of the functional tests, the `--` is used to pass arguments into the command